lib/core/db/PostgreSQLQueries.js

```diff
@@ -29,6 +29,12 @@ queries.createTables = [
         + "     earliest_parsed_id bigint, "
         + "     latest_unparsed_id bigint "
         + " ); ",
+    " CREATE TABLE IF NOT EXISTS stats "
+        + " ( "
+        + "     time timestamp with time zone, "
```

Comment:
with timezone? Do we want it to be zoned or should we not zone it and always make it UTC (or even just a bigint like the `sent_at` field in emotes)?

Reply:
Also I'd like a more significative name on this. "time" is a bit too generic. Maybe something like "created_at" or "recorded_at" (don't forget to edit the query if you change this).

Reply:
If I understand it correctly, `with time zone` stores timezone along with the date and time, which allows accurate date storage regardless of where the bot is hosted. Caring about which time zone is used is something we may want to do outside of the database.

Reply:
`created_at` is a better name for it, let's use that.